ci/libraries_tests/nrf5_app_sdcard/CMakeLists.txt

```diff
@@ -19,85 +19,50 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
+#
+# WARNING: FILE GENERATED FROM ./ci/scripts/generate_cmake_tests.sh SCRIPT.
```

Comment:
Hmm? What happened here?

Reply:
What do you mean? I assumed you add tests for libraries by running generate_cmake_script.sh, tweaking the generated CMakeLists.txt and adding main.c and sdk_config.h.

Reply:
I was confused about move operation. I saw that you removed one example and that's the reason.